src/Symfony/Installer/DemoCommand.php

```diff
@@ -26,7 +25,6 @@ class DemoCommand extends DownloadCommand
     protected $fs;
```

Comment:
You are right. I moved the initialization of the variables but not their declaration. Done. Thanks.